english/critter/critter.txt

```diff
@@ -0,0 +1,5 @@
+A critter approaches, wiggling its ears and sniffing, seeming to know that you have recently eaten a sandwich. Do you 
+
+[feed the critter](feed/feed.md) the other sandwich you had, hoping to make friends? Or
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.